tenderly-suite/role-update-action/role-update/roleUpdate.js

```diff
@@ -0,0 +1,102 @@
+const axios = require('axios').default;
+
+const TOPIC_0_ROLES_ADDED = '0x3df2f62906643352cfb366ada865850a1f2127a98c97ac962921d5caf75561c3';
+const TOPIC_0_ROLES_REMOVED = '0x17e90d13bc6dcdbe950d3d022f0774c9dfa3308b96720b8779075dd83236061f';
+
+const PAGER_DUTY_ALERT_URL = 'https://events.pagerduty.com/v2/enqueue';
+
+const TRUSTED_ADDRESSES = [
+    '0x4a6eea0999b000a941926e298f7a49373c153fbc', // TODO: Update contracts list upon deployment
+    '0xf1ea5615086a0936f82656f88263365831978f71',
+    '0xb8cd93c83a974649d76b1c19f311f639e62272bc',
+];
+
+const handleRoleUpdate = async (context, event) => {
+    const chainName = context.metadata.getNetwork();
+
+    const roleAddedAccounts = [];
+    const addedRoles = [];
+    const roleRemovedAccounts = [];
+    const removedRoles = [];
+    const summary = 'Roles updated';
+    let severity = 0;
+
+    for (const log of event.logs) {
+        if (log.topics[0] === TOPIC_0_ROLES_ADDED || log.topics[0] === TOPIC_0_ROLES_REMOVED) {
+            console.log('log Found');
+            const length = parseInt(log.data.substring(128, 130), 16);
```

Comment:
Maybe assign some of these numbers to variables or add a comment to give some more context on the math being performed here